### PR TITLE
20190325 release visualizer

### PR DESCRIPTION
**Pull request by Mulesoft-Mysti**
@asimons-mule and anyone else who want to review? Feel free to leave all comments--not sure if I overdid it with the app/server properties notes. RN coming shortly.

### CODE REVIEW

**Comment by Mulesoft-Mysti** [comment]
@asimons-mule please feel free to review even if merged. Worried about doc going out after release, so had @valkyrie69 review.

File: modules/ROOT/pages/setup.adoc [hunk 1]
```diff
@@ -71,6 +71,21 @@ Anypoint Visualizer supports the following Mule versions for standalone Mule dep
...
+However, due to a potential performance impact for Mule 4.1.x, header injection has been disabled by default,
+so you must enable it at the server, not application, level.
+
+Set the following property in the server's properties:
++
+----
+`anypoint.platform.config.analytics.agent.header_injection.disabled=false`
+----
+
+Note that you must set the property to `false` as shown in the example to enable header injection, since the property is `disabled`.
```
Comment (valkyrie69) [inline modules/ROOT/pages/setup.adoc]: Tag as NOTE if you think it's important enough.

File: modules/ROOT/pages/layers.adoc [hunk 2]
```diff
@@ -13,6 +13,7 @@ Map a node to a layer in the application's properties file, or use the Anypoint
 === Set a Layer with Properties
 
 To automate the classification of nodes into layers, configure the property `anypoint.platform.visualizer.layer=<name>` in the application's properties file.
+Note that this is an application property, not a server property.
```
Comment (valkyrie69) [inline modules/ROOT/pages/layers.adoc]: Tage with [NOTE] if you think it's important enough

File: modules/ROOT/pages/setup.adoc [hunk 3]
```diff
@@ -71,6 +71,21 @@ Anypoint Visualizer supports the following Mule versions for standalone Mule dep
 
 To support Visualizer for standalone Mule deployments, xref:monitoring::am-installing.adoc[install the Anypoint Monitoring agent]. 
 
+=== Enable Header Injection
+
+Visualizer requires an additional header injection to work properly.
+However, due to a potential performance impact for Mule 4.1.x, header injection has been disabled by default,
```
Comment (valkyrie69) [inline modules/ROOT/pages/setup.adoc]: This is a long sentence. I would prob break it up into two.

**Review by valkyrie69 — APPROVED** [review]
Looks good! A couple of comments. No show stoppers.

**Review by asimons-mule — COMMENTED** [review]
Looks good! Just a few notes.

CloudHub doesn't have server properties so I don't think we need to make the distinction in those sections.

Since header injection separately affects performance. We might want to keep it in the Reduce Potential performance ... section. To be honest, it's hard for me to get a sense of the full doc right now so if you think it's covered clearly elsewhere and would just be repetitive. Let me know!

**Comment by Mulesoft-Mysti** [comment]
Done!

**Pull request merged**